examples/regional-trade/csvw/flow-directions.csv

```diff
@@ -1,3 +1,3 @@
-Label,Notation,Parent Notation
-Export,export,
-Import,import,
+label,notation,parent_notation,sort_priority,description,top_concept_of,has_top_concept,pref_label
```

Comment:
Here we see the benefit of doing this. We didn't keep this file up-to-date as these extra columns were introduced.